merge/external_parser_setup.py

```diff
@@ -0,0 +1,10 @@
+
+class ExternalParserSetup:
+    is_set = False
+
+    @classmethod
+    def setup(cls):
+        if not cls.is_set:
+            cls.is_set = True
+            from clang.cindex import Config
+            Config.set_library_file("/usr/local/opt/llvm/lib/libclang.dylib")
```

Comment:
Me neither, but this is something we have already had before. Presumably it is the default path to the library. 

The only way to remove this hard code is to create a search for this binary, cash it somewhere, then use it. This was planed for the future as a part of making the whole product cross platform and easily deployable.

Summing up, fixing it is complicated and deserves its own PR. I propose we leave this hard coded for now and create an issue to fix it later.